archive/readme.md

```diff
@@ -38,8 +38,7 @@ Then add the cards into it.
 
 ## How does it work
 
-The `getArchiveCards()` method creates `<object>` elements. As many as there are archive files (based on the manually updated `NUMBER_OF_FILES`).  
-Each object element imports the archive file with the corresponding index.  
+The getArchiveCards() method creates an <object> element for each archived file. The total number of archived files is based on the manually updated NUMBER_OF_FILES.
```

Comment:
Hi @rahulsust, this has been a really good learning process for us both on code reviews 😆 . Before I can merge this it appears the backticks (`) were removed from getArchiveCards(), the object tag and the NUMBER_OF_FILES variable. Its important to add these in markdown files to have code snippets render correctly. If you could add those backticks back in that would be awesome!! Thank you 👍

Reply:
Hey @lax0248917, I overlooked those mistakes 😅.  I am enjoying working with you.  Thanks for your continuous support.